tap_helpscout/client.py

```diff
@@ -42,111 +49,108 @@ def __enter__(self):
     def __exit__(self, exception_type, exception_value, traceback):
         self.__session.close()
 
-    @backoff.on_exception(backoff.expo,
-                          Server5xxError,
-                          max_tries=5,
-                          factor=2)
     def get_access_token(self):
```

Comment:
Should we consider adding backoff while generating the access token?

Reply:
Added